lib/vdsm/storage/mpathconf.py

```diff
@@ -400,14 +400,10 @@ def check_conf_revision():
     Returns:
         bool: check has passed.
     """
-    first = second = ''
     with open(CONF_FILE) as f:
-        mpathconf = [x.strip("\n") for x in f.readlines()]
-    try:
-        first = mpathconf[0]
-        second = mpathconf[1]
-    except IndexError:
-        pass
+        # Read only the lines that are going to be processed
+        first, second = [x.strip("\n")
+                         for x in (f.readline() for _ in range(2))]
```

Comment:
This is much more complicated than needed. If you want to read the first 2 lines, just do:

    first = f.readline().rstrip("\n")
    second = f.readline().rstrip("\n")

Also we want to strip only the right side, so we should use rstrip().